### ISSUE TITLE
Network error

### ISSUE THREAD

**Issue by qaqland** [faldez/tanoshi#294]
My network has been blocked with some sites like github and google. So I can hardly use it. :(

## when use server
```
[2022-04-17T06:33:06Z ERROR tanoshi::worker::downloads] no queue
[2022-04-17T06:33:07Z ERROR tanoshi::worker::updates] failed check extension update: error sending request for url (https://raw.githubusercontent.com/faldez/tanoshi-extensions/repository/index.json): error trying to connect: tcp connect error: Connection refused (os error 111)
```

## when use local appimage

![图片](https://user-images.githubusercontent.com/62464571/163703662-d8303c04-860f-4560-baa7-ee62596d85d1.png)

I think it should can be use first without checking network.

My ENV:
- OS: Manjaro Linux x86_64
- Kernel: 5.15.28-1-MANJARO
- DE: GNOME 41.5 
- NETWORK: China(offline)

Best Wishes/祝好

**Comment by faldez** [comment]
hmm let me check, extension update check shouldn't affect functionality

**Comment by faldez** [comment]
v0.28.0 tauri build is broken somehow. on v0.27.1 works fine. will fix next release

**Comment by faldez** [comment]
fixed on [v0.28.0](https://github.com/faldez/tanoshi/releases/tag/v0.28.1)
